scene/gui/graph_edit.cpp

```diff
@@ -2417,18 +2418,18 @@ GraphEdit::GraphEdit() {
 	top_layer->connect("focus_exited", callable_mp(panner.ptr(), &ViewPanner::release_pan_key));
 
 	connections_layer = memnew(Control);
+	connections_layer->set_name("_ConnectionsLayer");
 	add_child(connections_layer, false, INTERNAL_MODE_FRONT);
 	connections_layer->connect("draw", callable_mp(this, &GraphEdit::_connections_layer_draw));
-	connections_layer->set_name("CLAYER");
```

Comment:
Also, what about this node? If we were to remove the names, there's also this internal node which already has a name in master. Same with the scroll bar nodes. If we don't desire these nodes to have names, I guess we should remove the names of these nodes too for consistency. Anyway, it should be changed regardless, `_ConnectionsLayer` is more readable than `CLAYER`.